picasso-compose/src/main/java/com/squareup/picasso3/compose/PicassoPainter.kt

```diff
@@ -40,14 +40,21 @@ fun Picasso.rememberPainter(
   request: (Picasso) -> RequestCreator
 ): Painter {
   return remember(key) { PicassoPainter(this, request, onError) }
```

Comment:
It's not strictly related to this change but there's another bug here: `request` and `onError` should either be passed as keys to `remember`, stored in `rememberUpdatedState`, or updated on the painter on recomposition. E.g. if this recomposes while the load is in progress and the caller passes in a new `onError` lambda, the new handler will be ignored.